code/modules/client/preference/link_processing.dm

```diff
@@ -476,6 +476,9 @@
 
 							valid_markings += markingstyle
 						sortTim(valid_markings, GLOBAL_PROC_REF(cmp_text_asc))
+						if(length(valid_markings) < 1) // Some IPC head models do have head markings, some don't; This is here to prevent us from attempting to open an empty TGUI list
```

Comment:
```suggestion
						if(!length(valid_markings)) // Some IPC head models do have head markings, some don't; This is here to prevent us from attempting to open an empty TGUI list
```